static/src/javascripts/__flow__/types/ab-tests.js

```diff
@@ -59,6 +59,7 @@ declare type EpicABTest = AcquisitionsABTest & {
 declare type InitEpicABTestVariant = {
     id: string,
     products: $ReadOnlyArray<OphanProduct>,
+    test?: (html: string, abTest: ABTest) => void,
```

Comment:
This replaces the `options.test` in the `Variant` type. If it's not supplied then the standard epic insertion logic is used (in `makeABTestVariant`).